cli/types/index.d.ts

```diff
@@ -2458,7 +2458,13 @@ declare namespace Cypress {
     experimentalGetCookiesSameSite: boolean
   }
 
-  interface PluginConfigOptions extends ConfigOptions {
+  /**
+   * When users specify config options, they can provide only
+   * some properties.
+   */

```

Comment:
The way that this is phrased implies that users cannot supply all config options, maybe something like this:

```suggestion
  /**
   * All configuration items are optional.
   */
```